spark/README.md

```diff
@@ -135,12 +135,86 @@ echo "match (a) -[r] -> () delete a, r;match (a) delete a;" | cypher-shell -u ${
 ```
 
 Then run the example:
+
 ```bash
 scripts/run-graphar2neo4j.sh
 ```
 
 The example will import the movie graph from GraphAr to Neo4j and you can check the result in the Neo4j browser.
 
+## Running NebulaGraph to GraphAr example
```

Comment:
The Nebula2GraphAr example needs docker, we need to proved a hint here to let users know they should install docker first